internal/stage_common.go

```diff
@@ -132,7 +132,7 @@ func validateContent(resp http.Response, expectedContent string, expectedContent
 	}
 
 	if string(body) != expectedContent {
-		return fmt.Errorf("Expected the content to be %s got %s", expectedContent, body)
+		return fmt.Errorf("Expected the content to be %q got %q", expectedContent, body)
```

Comment:
let's refactor this to be
```go
logger.Errorf("Expected content: %q", expectedContent)
logger.Errorf("Received content: %q", string(body))
return fmt.Errorf("Content mismatch")
```
This will be easier to parse for the end user.

Reply:
Feel free to ignore this, in this PR. I can pick it up later.